### DISCUSSION TITLE
Source of @librechat/agents?

### DISCUSSION THREAD

**Discussion by alex-torregrosa** [danny-avila/LibreChat#5043] [Help Wanted]
While trying to tinker with the Agents code to add support for Gemini models, I found out that a lot of logic depends on the `@librechat/agents` package, which I am unable to find anywhere other than NPM. Shouldn't it be on this repo?

**Comment by danny-avila** [comment]
No, I am managing this privately. I will add Google support soon.

**Comment by rajvatsal** [comment]
What's the point of open source if it's not 100% open source?

**Reply by danny-avila** [reply]
It is open source, it was private while in early development: https://github.com/danny-avila/agents

**Reply by rajvatsal** [reply]
My bad. Searching led me here.
